ethereum_history_api/oracles/fixtures/mainnet/london/crypto_punks/eth_getProof_14194126.json

```diff
@@ -1,7 +1,11 @@
 {
   "method": "getProof",
   "arguments": [
-    { "address": "0xb47e3cd837ddf8e4c57f05d70ab865de6e193bbb", "storageKeys": [], "blockNumber": 14194126 }
+    {
+      "address": "0xb47e3cd837dDF8e4c57F05d70Ab865de6e193BBB",
```

Comment:
Viem is returning checksummed addresses now, had to change them in few places as mocked calls were not working due case sensitive comparisons